x-pack/legacy/plugins/ml/public/license/check_license.tsx

```diff
@@ -4,51 +4,45 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-
 import React from 'react';
+// @ts-ignore No declaration file for module
+import { banners } from 'ui/notify';
```

Comment:
Is there going to be a NP replacement for `banners` in `ui/notify`? If not, could we just use a callout?

Reply:
i'm not sure if banners will be replaced or not.
This could be switched for a callout but we'd need to create a way of copying the banner behaviour where it can be force added to every page.
I'd suggested this investigation and work be done at a later stage